cli/validate-wca.js

```diff
@@ -0,0 +1,66 @@
+import chalk from 'chalk';
+import { fileURLToPath } from 'node:url';
+import path from 'node:path';
+import { readFile } from 'node:fs/promises';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+const customElementsPath = path.join(__dirname, '../custom-elements.json');
+
+function validateTag(tag) {
+
+	// assume an element with no description is to be ignored
+	if (!tag.description) {
+		return 0;
+	}
+
+	let errors = 0;
+
+	tag.attributes?.forEach(a => {
+		if (!a.description) {
+			console.log(`Attribute "${a.name}" on tag "${tag.name}" is missing a description.`);
+			errors++;
+		}
+	});
+
+	tag.events?.forEach(e => {
```

Comment:
Should we be checking that all slots have a description too?

Reply:
I was going to say "no, the list of slots come from the comments" but even though that _is_ true, they could technically still go `@slot foo` and not provide a description so I'll add a check for it!

Reply:
Done!